src/components/Messages.vue

```diff
@@ -98,13 +85,22 @@ export default {
       const container = this.$el.querySelector('#container')
       container.scrollTop = container.scrollHeight
     },
-    loadMoreMessages () {
-      this.lastSeenMsgId = this.messages[0].entry.uuid
-      this.getMessageChunk({
+    scrollToMessage (id) {
+      if (!id) return
+      const container = this.$el.querySelector('#container')
+      container.scrollTop = 0
+      const messageElement = document.getElementById(id)
+      const offset = messageElement.getBoundingClientRect().top - messageElement.offsetParent.getBoundingClientRect().top
+      container.scrollTop = offset - 100
```

Comment:
without it, the message you scroll to appears at the very top of the page, which is weird UX after you've just clicked load more messages. The UX is: I clicked "load more messages", and now the "load more messages" button is gone but I'm just seeing the same messages I was looking at.
the `- 100` means that the message you scroll to stays in the same spot on the screen but the "load more messages" button is replaced with the next couple of earlier messages

Reply:
Maybe explain that in a comment? "Scroll up by 100px so that space taken by Load more messages button is replaced with actual messages" Or maybe that 100px should be derived in `loadMoreMessages` by taking the offset of the top message prior to clicking load more messages?

Reply:
> Why is there a new file at `tests/integration/setup/bundle/elemental-chat-develop.happ`?

just something I kept around for testing. Removed